Explorer/Assets/DCL/Profiles/GetProfileIntention.cs

```diff
@@ -0,0 +1,38 @@
+using Arch.Core;
+using ECS.StreamableLoading.Common.Components;
+using System;
+using System.Threading;
+
+namespace DCL.Profiles
+{
+    public struct GetProfileIntention : ILoadingIntention, IEquatable<GetProfileIntention>
+    {
+        public string ProfileId { get; }
+        public int Version { get; }
+        public EntityReference Entity { get; }
```

Comment:
You should not reference an entity if you are not gonna use it in the corresponding implementation of `LoadSystemBase`.
Instead, you should incorporate this promise in the entity that corresponds to the Player/Avatar